libs/data-mapper/src/lib/models/Schema.ts

```diff
@@ -53,6 +53,11 @@ export enum NormalizedDataType {
   String = 'String',
 }
 
+export enum InputFormat {
```

Comment:
Should we also have `Dropdown` as an option so that we are explicit about which input? Or just always default to dropdown?

Reply:
Default is dropdown. Discussed this with Sanjiv yesterday! And if we want we can add more in the future